examples/active-class-name/next.config.js

```diff
@@ -1,3 +1,4 @@
+/** @type {import('next').NextConfig} */
```

Comment:
I think we need to assign the config to a variable first before exporting for the types to apply correctly, this is why our default template does this

Reply:
Afaik this issue has been solved. It does not make a difference if you assign to a variable (`const` | `let`) or to an object property (`module = {}`).

Check out the following [CodeSandbox](https://codesandbox.io/s/youthful-archimedes-8czs7i?file=/next.config.js) to test this yourself.

If you use "decorator" functions (`withFoo({})`, `withTranspiledModules({})`, etc.) assigning to a variable makes sense.

@ijjk Please let me now if you still want to rely on this convention. In this case I will adjust the PR.

Reply:
Check out the tailwind example to see other projects following this "new" convention: https://tailwindcss.com/docs/installation

Reply:
Ah nice it does look like this is resolved in newer TypeScript versions